help/en/html/apps/DecoderPro/CreateDecoderAdvanced.shtml

```diff
@@ -694,6 +694,10 @@ delay is 500 milliseconds.
       written for each operation, in which case this should be set to false. The default is
       true.</p>
 
+      <p>In some cases, you want to reset the PI and SI values after the indexed CV
+      is read or written.  To do that, put the new values after a semicolon:
+      11.12.123;0.1 for example
```

Comment:
What happens if the decoder definition uses the format PI.CV? It seems to me that this doesn't work in that case. I have no problem with that, but if so, it might be good to mention that this feature only works for when both PI and SI are in use, and not when only PI is in use.